source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+As it turned out setting up multiple hosts at once can be rather tedious via
+UI. The slow point is that the administrator has to go through every new host
+and do Setup Host Network manually. By utilizing Copy Host Networking, the administrator
+has to set only single host, for the rest the administrator can just specify the host
+to take the network configuration from. 
+
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
+
+The Copy Host Network helps the administrator with Day 1 deployment automation
+for host networking configuration across multiple hosts. This feature can also be
+on Day 2 for any additional host that needs the same configuration of networking. 
+
+### Example flow
+
+After creation of new Cluster, the administrator adds several hosts (`Host0, Host1, Host2`). 
+`Host0` networking has to be configured manually either by UI or through REST API.
+Other hosts, as long as they have same layout of network interfaces, can copy
+their settings from the host that has been already configured. 
+
+The copy can be done via UI. Going to hosts network interfaces and by clicking
+the `Copy Host Networking` button. In the popup window that will appear 
+the administrator can select from which host (`Host0`) should the configuration be copied.
+
+Via REST API every host has method called `copynetworks` which needs a single
+parameter, id of the host (`Host0`) that the configuration should be copied from. 
+
+### Limitation
+
+* The host configuration can be copied only within the same cluster. 
+
+* Configuration of static IP address for any of the networks cannot be copied. The static IP
+networks have their IP incremented by one to stay in range of the mask. 
+
+* Host with same interface names but different physical network connections will result in bad configuration. 
```

Comment:
what bad configuration is means? does it just fail to copy networks?
what happen for add new host scenario? does the add host should fail?

Reply:
Everything will be copied but the logical structure of the network will be messed up. But there is nothing from our side we can do about this. It is up to the administrator that manages the deployment.

Reply:
What will happen during add host? the host will be added or the admin will need to fix his host or just not choose to copy host networks?

Reply:
The operation should pass, it is not blocker in any way. It is there just to inform the administrator that it might happen.